lessons/pydata/homework_revisited/index.ipynb

```diff
@@ -0,0 +1,370 @@
+{
```

Comment:
<p>Slovo "evaluační" slyším poprvé. Nepoužívá se spíš "validační"? (Jakkoliv evaluace je lepší popis toho, co děláme)</p><p> </p><p>(berte s rezervou, nejsem teoretik ML)</p><p> </p><p>P.S. Nějak mi z toho odstavce nepřijde zřejmé, že jsem teď celou dobu zacházel s testovací množinou špatně a mám to začít dělat líp. Nedalo by se v tom být nějak jasnější a ostřejší?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>